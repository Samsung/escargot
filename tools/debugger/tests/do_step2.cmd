step
s
s
s
s
s
s
s
s
s
s
s
