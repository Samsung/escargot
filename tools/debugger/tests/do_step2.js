/*
 * Copyright (c) 2020-present Samsung Electronics Co., Ltd
 *
 *  This library is free software; you can redistribute it and/or
 *  modify it under the terms of the GNU Lesser General Public
 *  License as published by the Free Software Foundation; either
 *  version 2.1 of the License, or (at your option) any later version.
 *
 *  This library is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 *  Lesser General Public License for more details.
 *
 *  You should have received a copy of the GNU Lesser General Public
 *  License along with this library; if not, write to the Free Software
 *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
 *  USA
 */

var arr = (x) => x + x
arr();

var a;
var b,
    c;

var d = 1;
var e = 2,
    f = 3;

let g;
let h,
    i;

let j = 4;
let k = 5,
    l = 6;

const m = 7;
const n = 8,
      o = 9;
