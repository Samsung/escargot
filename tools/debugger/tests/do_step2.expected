Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_step2.js:20
(escargot-debugger) step
Stopped at tools/debugger/tests/do_step2.js:21
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:20 (in arr() at line:20, col:11)
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:27
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:28
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:29
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:35
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:36
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:37
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:39
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:40
(escargot-debugger) s
Stopped at tools/debugger/tests/do_step2.js:41
(escargot-debugger) s
Connection closed.
